Prepare for 1.9.1 (#597)
microsoft/vscode-cordova: CHANGELOG.md

## 1.9.1

> * Added error notifications about usage of incorrect `cwd` parameter when trying to debug a project located in a subdirectory [#593](https://github.com/microsoft/vscode-cordova/pull/593)

Reviewer: ```suggestion
* Added error notifications about usage of incorrect `cwd` parameter on debug a project located in a subdirectory [#593](https://github.com/microsoft/vscode-cordova/pull/593)
```

* Updated troubleshooting documentation for the issue [#574](https://github.com/microsoft/vscode-cordova/issues/574) ([#575](https://github.com/microsoft/vscode-cordova/pull/575)), thanks to [David Cox-Espenlaub(@newdaveespionage)](https://github.com/newdaveespionage)

> * Fixed security vulnerabilities

Reviewer: ```suggestion
* Enhanced extension security
```
